helm-charts/flink-historyserver/templates/historyserver.yaml

```diff
@@ -0,0 +1,98 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: historyserver
+  name: historyserver
+spec:
+  replicas: 1
+  revisionHistoryLimit: 10
+  selector:
+    matchLabels:
+      app: historyserver
+  template:
+    metadata:
+      labels:
+        app: historyserver
+    spec:
+      containers:
+      - args:
+        - history-server
+        command:
+        - /docker-entrypoint.sh
+        env:
+        - name: _POD_IP_ADDRESS
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: status.podIP
+        image: flink:{{- .Values.flinkVersion }}
+        name: flink-main-container
+        ports:
+        - containerPort: 8082
+          name: history
+          protocol: TCP
+        resources:
```

Comment:
This should be configurable instead, and these values can be the default there in values.yaml file if you want.

Reply:
added to values